gradle.properties

```diff
@@ -66,3 +66,6 @@ bookkeeperBaseTag=pravega/bookkeeper
 signing.keyId=
 # This will be defaulted to ~/.gnupg/secring.gpg if not provided as a command line property
 signing.secretKeyRingFile=
+
+enableMkdocs=true
```

Comment:
Is there a reason why it was disabled in the older build?

Reply:
It was enabled by default initially (see PR #2169) but then your question made me checking the history of some other files and it seems there was an issue with building Flink Connector (see Issue #2423).  @crazyzhou  is helping to check whether the issue is still present if `mkdocs`  plugin is enabled by default or not.